src/Dataflow/Engine/Controller/PythonImpl.cc

```diff
@@ -613,7 +615,7 @@ boost::shared_ptr<PyModule> PythonImpl::findModule(const std::string& id) const
 
 std::string PythonImpl::executeAll(const ExecutableLookup* lookup)
 {
-  cmdFactory_->create(GlobalCommands::DisableViewScenes)->execute();
+  //cmdFactory_->create(GlobalCommands::DisableViewScenes)->execute();
```

Comment:
Temporary? Or not needed?

Reply:
This was a bug fix that might not be needed anymore, but needs testing on other major platforms, which I haven't been able to build yet. So it's in the netherworld of "commented out code hopefully soon to be deleted"--which I should use a special symbol for or something.

Reply:
What about 'TODO DELETE if X works'?

Reply:
Yeah, there's a lot of commented code that fits that category--lots of work to reference the correct issue though...#lazyprogrammer